### PR TITLE
[Reviewer: Graeme] Minor fixes

### PR DESCRIPTION
**Pull request by rkday-pro**
Various small fixes I've made based on feedback from you and/or Ellie, or my own testing:

* Rename to `clearwater-sip-stress-coreonly` so it can coexist with the old stress scripts
* Make the re-register rate per second a float, so it can be something like 0.3 - previously it was an integer (so if it was <1 it got truncated to 0)
* Allow the I-CSCF and S-CSCF to be specified separately
* Don't send a BYE when a REGISTER flow fails

I'll test before merging.

### CODE REVIEW

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress [hunk 1]
```diff
@@ -117,49 +121,54 @@ with open(INVITE_CSV, "w") as f:
 null = open("/dev/null", "w+")
 
 common_args = ["-key", "home_domain", args.domain,
-               "-t", "t1",
-               args.target]
+        "-t", "t1"]
```
Comment (graemerobertson) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: Noooooooooooooooo!

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress [hunk 2]
```diff
@@ -117,49 +121,54 @@ with open(INVITE_CSV, "w") as f:
...
-                   "-inf", REGISTER_CSV,
-                   "-trace_err", "-error_file", INITIAL_REG_ERRORS,
-                   "-r", str(args.initial_reg_rate),
-                   "-m", str(args.subscriber_count)] + common_args
+        "-sf", SCRIPTS_DIR + "register.xml",
+        "-inf", REGISTER_CSV,
+        "-trace_err", "-error_file", INITIAL_REG_ERRORS,
+        "-default_behaviors", "all,-bye",
+        "-r", str(args.initial_reg_rate),
+        "-m", str(args.subscriber_count)] + common_args + [args.icscf_target]
```
Comment (graemerobertson) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: These realignments may be deliberate?! I don't like it, I don't know why it's happened.

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress [hunk 3]
```diff
@@ -198,12 +207,12 @@ print "Test complete"
...
 with open(CALLER_STATS) as f:
-	r = DictReader(f, dialect=SIPpstats())
-	for row in r:
-		pass
-	hours, mins, secs, nanoseconds = map(int, row['ResponseTime1(C)'].split(":"))
-	rtt = (nanoseconds / 1000.0) + (secs * 1000.0)
-	print """
+    r = DictReader(f, dialect=SIPpstats())
+    for row in r:
+        pass
```
Comment (graemerobertson) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress:212]: I've probably in theory reviewed this code before, and I know you haven't changed it, but I have no idea what's going on here.

**Review by graemerobertson — APPROVED** [review]
Is there a corresponding SIPp change for this BYE stuff? Anyway, looks fine.

**Comment by rkday-pro** [comment]
Nope, the BYE stuff is standard SIPp function (sipp.sourceforge.net/doc/reference.html).

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress (hunk 2 above)
Comment (rkday-pro) [reply clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: That's what vim does when you use `=G` to re-indent the whole file. I've fixed.

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress (hunk 3 above)
Comment (rkday-pro) [reply clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress:212]: I've added a comment (it's to get the last row).

**Pull request merged**